app/policies/article_policy.rb

```diff
@@ -52,11 +81,16 @@ def permitted_attributes
   end
 
   def subscriptions?
+    require_user!
     user_author? || user_super_admin?
   end
 
   private
 
+  def require_user!
+    raise Pundit::NotAuthorizedError, I18n.t("policies.application_policy.you_must_be_logged_in") unless user
```

Comment:
this is just a personal preference but I like the explicit return early
```suggestion
    return if user
    raise Pundit::NotAuthorizedError, I18n.t("policies.application_policy.you_must_be_logged_in")
```

Reply:
Definitely like guard conditions, and agree that this is a bit nicer